src/components/bridge/ConnectSmall.vue

```diff
@@ -0,0 +1,313 @@
+<template>
```

Comment:
The `ConnectSmall` component works well for switching between mobile and desktop view, but unfortunately this solution results in a lot of repeated code. We will have to see if we can just stick with a single `Connect` component. 

I can think of two directions that might work, but you can do whatever works.
- Do the check for screensize in the `Connect` component itself.
- Or use flexbox in a clever manner. The div in the parent component (`bridgeStepper`) that wraps `Connect` will be the flex container for `Connect`, which is the flex item.